docs/src/parameter_distributions.md

```diff
@@ -449,6 +463,51 @@ using Plots
 plot(u, constrained = false)
 ```
 
+## [Function Distribution Example](@id function-example)
+Here, define a function parameter distribution on ``[0,1] \times [1,2]`` , bounded by ``[-5,-3]`` and with correlation lengthscales 0.05. First, we get the packages:
+
+```@example snip_fun
+using EnsembleKalmanProcesses.ParameterDistributions # For `ParameterDistribution`
+using Random, Distributions # for `rand` and `Normal`
+using Plots
+# IMPORT the GRF package
```

Comment:
I suspect you're trying to say use `import` instead of `using` to load GRF. I think making that explicit could help readability.

Reply:
Done